cmd/tsbs_generate_data/devops/mem.go

```diff
@@ -60,17 +59,21 @@ func (m *MemMeasurement) ToPoint(p *serialize.Point) {
 	p.SetTimestamp(&m.timestamp)
 
 	total := m.bytesTotal
-	used := m.distributions[0].Get()
-	cached := m.distributions[1].Get()
-	buffered := m.distributions[2].Get()
+	used := int64(math.Floor(m.distributions[0].Get()))
+	cached := int64(math.Floor(m.distributions[1].Get()))
```

Comment:
Do we need `math.Floor()` here? Won't casting a float to an int always do the same thing? Or is this one of those weird things where floating point arithmetic sometimes gets weird?

Reply:
afaik its only different for negative values